examples/nvidia-smi-gpu-monitoring.yml

```diff
@@ -6,6 +6,13 @@
 # issue with memory temperature returning NA
 # https://forums.developer.nvidia.com/t/request-gpu-memory-junction-temperature-via-nvidia-smi-or-nvml-api/168346/118
 #
+# on windows the infra-agent may not have environment variables reqiured for newer versions of nvidia-smi to run.  Currently, the three required appear to be:
```

Comment:
Could this be related with the fact that the environment variables that flex requires need to be passed by the infrastructure-agent? The way to specify them is to use this configuration option in newrelic-infra.yml
https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings/#passthrough_-environment

Reply:
Added the link into the comment - also updated troubleshooting agent to have the link to that section of the docs.  

It might also be a good idea to point to this directly in the agent installation steps.